tests/expected.out

```diff
@@ -52,7 +52,6 @@
 /tests/data/sql-set-value.php:14:Value 'doesNotExist' does not exist in table selected via setTable().
 /tests/data/sql-set-value.php:22:Value 'doesNotExist' does not exist in table selected via setTable().
 /tests/data/sql-set-value.php:24:setArrayValue() expects a database column which can store string values.
-/tests/data/sql-set-value.php:38:Value 'doesNotExist' does not exist in table selected via setTable().
```

Comment:
we refrain on this error which is only possible when allowing multi-constant unions.

I think the primary use case of this PR is way more important then maybe-union table name